--- src/setup_ann.py	(revision 442)
+++ src/setup_ann.py	(working copy)
@@ -29,8 +29,8 @@
 
 libann_wrap = Extension(name = '_libann_wrap',
                 sources = ['ann/ann_wrap.cpp'],
-                include_dirs=[os.path.join(GetANNDirectroyFromROS(), 'include')],
-                library_dirs=[os.path.join(GetANNDirectroyFromROS(), 'lib')],
+                include_dirs=[str(os.path.join(GetANNDirectroyFromROS(), 'include'))],
+                library_dirs=[str(os.path.join(GetANNDirectroyFromROS(), 'lib'))],
                 libraries=['ANN_1_1_1'])
